### PR TITLE
Update on Coseismic using staged data

### PR DESCRIPTION
**Pull request by nabolfat**
Coseismic: added lines to be able to read config files from all staged data.

### CODE REVIEW

**Comment by dbekaert** [comment]
I notived as well that it might perhaps be better if we do not label the appendix with a number. or rather use A1, B1 etc. A reviewer might just be scrolling down completely to the end and miss the results in like section 6 as being the final?

**Comment by EJFielding** [comment]
<p>Yes, I had added </p><pre class="ql-syntax" spellcheck="false">&lt;div class="alert alert-warning"&gt;
Final result Method 2—
&nbsp;&nbsp;68% of points below the requirements line is success
&lt;/div&gt;
</pre><p>It somehow got lost in this version.</p>

---

 <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/35/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by yunjunz** [comment]
GitHub flagged conflicts in the "Secular_*" notebook. Could you rebase your branch against the latest upstream/nisar-solid main branch?

**Comment by EJFielding** [comment]
<p>@nabolfat can you add that back to the notebook here?</p><p> </p>

---

 <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/35/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: methods/coseismic/Coseismic_Requirement_Validation.ipynb [hunk 1]
```diff
@@ -96,11 +96,11 @@
     "- [6.3. Amplitude vs. Distance of Relative Measurements (pair differences)](#M2ampvsdist2)\n",
```
Comment (EJFielding) [inline methods/coseismic/Coseismic_Requirement_Validation.ipynb:96]: <p>@nabolfath Can you change this to:</p><pre class="ql-syntax" spellcheck="false">print('\n site&nbsp;&nbsp;disp-los [cm]')
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

**Comment by nabolfat** [comment]
I rebased my last 3 commits. 
Could you please verify that the conflict is solved.

**Comment by nabolfat** [comment]
I rebased my last 3 commits. I have made ready two notebooks including all the changes for Secular and Coseismic. @yunjunz can you please verify that the conflict is resolved and it is ok for me to add new commits?

**Comment by yunjunz** [comment]
@nabolfat I am gonna re-open this PR so that I could run `git rebase` on your branch.

**Comment by yunjunz** [comment]
@nabolfat Github conflict is resolved now.

**Comment by dbekaert** [comment]
<p>This is still to be adressed</p>

---

 <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/35/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by dbekaert** [comment]
@EJFielding Remaining comments are minor, but could have a large impact to how a reviewer experiences the process.
I  don't think the take-aways are very clear.
We don't explain what the final result tables shows (e.g. variance in xx units as function of distance in km), or have cells printed without description (e.g. we take about percentage but then we print fraction). Perhaps we can think of some sort of caption for them that gets printed as part of the code cell?  

We should also highlight the final criteria or success value or the rule. Also rules work of percentages and not fraction so suggest to emphasize fraction first then relate to percentage

**Comment by EJFielding** [comment]
<p>We added the alert but it is not shown in ReviewNB.</p>

---

 <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/35/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Comment by EJFielding** [comment]
<p>@dbekaert the boxes are there in the notebook, but ReviewNB is not showing them.</p>

---

 <a href='https://app.reviewnb.com/nisar-solid/ATBD/pull/35/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by EJFielding — APPROVED** [review]

**Pull request merged**